CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+			ideology = { factor = 0.04 value = progressive } 
+			ideology = { factor = 0.06 value = socialist } 
+			ideology = { factor = 0.04 value = communist } 
+			ideology = { factor = 0.02 value = communist_social } 
+			scaled_consciousness = { ideology = socialist factor = -9 }
+			scaled_militancy = { ideology = socialist factor = -9 }
+			scaled_consciousness = { ideology = communist factor = -9 }
+			scaled_militancy = { ideology = communist factor = -9 }
+			scaled_consciousness = { ideology = communist_social factor = -9 }
+			scaled_militancy = { ideology = communist_social factor = -9 }
+			consciousness = -2
+			militancy = -4
+		}
+		
+		ai_chance = { factor = 0.65 }
+	}
+	
+	option = {
+		name = "Legalize only non-communist parties"
+		badboy = -2
+		
+		prestige = 10
+		
+		any_pop = { 
+		    ideology = { factor = 0.02 value = big_tent } 
+			ideology = { factor = 0.07 value = liberal } 
+			ideology = { factor = 0.03 value = progressive } 
+			ideology = { factor = 0.08 value = socialist }  
+			scaled_consciousness = { ideology = socialist factor = -6 }
+			scaled_militancy = { ideology = socialist factor = -6 }
+			scaled_consciousness = { ideology = communist factor = 4 }
+			scaled_militancy = { ideology = communist factor = 4 }
+			scaled_consciousness = { ideology = communist_social factor = 4 }
+			scaled_militancy = { ideology = communist_social factor = 4 }
+			consciousness = -2
+			militancy = -2
+		}
+		
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 government = presidential_dictatorship } is_greater_power = yes }
+			relation = { who = THIS value = 50 }
+        }
+		
+		ai_chance = { factor = 0.30 }
+	}
+	
+	option = {
+		name = "Keep the parties as they are"
+		badboy = 3
+		
+		prestige = -25
+		
+		any_pop = { 
+		    ideology = { factor = 0.01 value = nationalist } 
+			ideology = { factor = 0.03 value = conservative } 
+			scaled_consciousness = { ideology = socialist factor = 4 }
+			scaled_militancy = { ideology = socialist factor = 4 }
+			scaled_consciousness = { ideology = communist factor = 4 }
+			scaled_militancy = { ideology = communist factor = 4 }
+			scaled_consciousness = { ideology = communist_social factor = 4 }
+			scaled_militancy = { ideology = communist_social factor = 4 }
+			consciousness = 2
+			militancy = 2
+		}
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 government = presidential_dictatorship } is_greater_power = yes }
+			relation = { who = THIS value = 75 }
+        }
+		
+		clr_country_flag = ITA_Democratic_Opposition
+		clr_country_flag = ITA_Democracy_Elections2
+		set_country_flag = ITA_Regime_coninuity
+		set_country_flag = ITA_Democracy_Elections
+		ai_chance = { factor = 0.05 }
+	}
+	
+}
+
+#Elections 4
+country_event = {
+ 	id = 8020417
+  	title = "$YEAR$ Italian General Election"
+ 	desc = EVT_8020417_DESC
+  	picture = "ITA_1953_elections"
+
```

Comment:
This event must have fire_only_once = yes

Reply:
See above